examples/advanced-blog/src/app/(web)/[...slug]/page.tsx

```diff
@@ -22,9 +22,21 @@ export async function generateMetadata(params: Params): Promise<Metadata> {
   const { doc, moreDocs } = await getData(params)
 
   if (!doc) {
-    return {
-      title: `All ${moreDocs.collection}`,
-      description: `All ${moreDocs.collection}`
+    // Check if moreDocs is an array
+    if (Array.isArray(moreDocs)) {
```

Comment:
More docs is only an array when doc is not `undefined`. So inside the`if (!doc)` statement `moreDocs` will always be an object. If you've found a case where we have `doc as an object` please let me know.